src/ContaoCommunityAlliance/DcGeneral/Contao/Dca/Populator/BackendViewPopulator.php

```diff
@@ -159,6 +160,11 @@ protected function populatePanel(EnvironmentInterface $environment)
                 }
             }
         }
+
+        $baseConfig    = $environment->getBaseConfigRegistry()->getBaseConfig();
+        $listingConfig = $viewDefinition->getListingConfig();
+
+        ViewHelpers::initializeSorting($panel, $baseConfig, $listingConfig);
```

Comment:
Could move it to an listener for the action event with a high priority instead. Any other solutions?